presentation-timestamps.md

```diff
@@ -46,3 +46,83 @@ This would keep compatibility with what's out there today, allowing progressive
 
 When exposing paint timings, we look for the right trade-off between "UX-precise" and "interoperable".
 By exposing those as two timestamps, and making one of them optional, we give web developers the information that can help them optimize, without compromising on interoperability.
+
+## Security & privacy self review 
+
+See [Self-Review Questionnaire: Security and Privacy](https://w3ctag.github.io/security-questionnaire/)
+
+### 01. What information might this feature expose to Web sites or other parties, and for what purposes is that exposure necessary?
+
+It exposes timing information of a platform/OS operation, namely "VSync".
+
+#### 02. Do features in your specification expose the minimum amount of information necessary to enable their intended uses?
+
+Yes.
+
+#### 03. How do the features in your specification deal with personal information, personally-identifiable information (PII), or information derived from them?
+
+This feature does not deal with personal information.
+
+#### 04. How do the features in your specification deal with sensitive information?
+
+This feature does not deal with sensitive information.
+
+#### 05. Do the features in your specification introduce new state for an origin that persists across browsing sessions?
+
+No. This feature only applies to the current document.
+
+#### 06. Do the features in your specification expose information about the underlying platform to origins?
+
+To some extent, the timing of committing a frame is information about the underlying platform.
+However, this information is already exposed in other ways (the `requestAnimationFrame` callback timestamp),
+and in this specification it is over-coarsen to avoid exposing meaningful information in terms of security.
```

Comment:
Did you mean "over-coarsened"? (And if so, does that imply that it's even more coarsened that necessary? Could this sentence just say "...it is coarsened to avoid exposing..."?)